src/main/java/edu/wisc/my/portlets/feedback/beans/Feedback.java

```diff
@@ -67,6 +67,11 @@
     private String profile;
     private String referrer;
     
+    //Customizable Links
+    private String chatLink;
+    private String callLink;
+    private String howToLink;
+    
```

Comment:
Interesting that we are injecting the links into the feedback object. This totally works, but i'm curious if we should instead just inject those links into the response model. Since its really part of the display model, not the feedback. Curious of others thoughts on this.